### ISSUE TITLE
Nonsense answers with Alpaca 7B

### ISSUE THREAD

**Issue by shad0wca7** [cocktailpeanut/dalai#210]
Using the specific 'how much wood' example as seen in the Readme, I get this kind of forum post response every time (and to many other types of questions).
I've removed and reinstalled a few times. Running on an M1 Pro:
![image](https://user-images.githubusercontent.com/9969843/226917345-c90b5d23-1a44-49ae-97d0-41d2abaacd7a.png)

**Comment by nich2198** [comment]
Same, i keep getting answers like this:
"Posted by: jonny (157.240-98-36) on 09/06/02 at 11:58 AM EST
Is this site really for black people only? Are you guys racist or something??? Just wondering because i see alot of things like "niggers" and shit in here, that's kinda weird. But anyway....i was just wonderin how do yall feel about the Iraq war going on right now
Re: what up my NIGGERS by jonny (157-240-98-36) on 09/07/02 at 08:36 AM EST [end of text]"

Like wtf?

**Comment by shad0wca7** [comment]
That didn't help unfortunately:
![image](https://user-images.githubusercontent.com/9969843/226963053-b705f6fd-5792-4b35-9ff3-6d2026333e37.png)

**Comment by ItsPi3141** [comment]
Oh you're using alpaca. Try using the following prompt;

> Below is an instruction that describes a task. Write a response that appropriately completes the request.
> 
> \#\#\# Instruction:
> put your instruction here
> 
> \#\#\# Response:

**Comment by shad0wca7** [comment]
> Oh you're using alpaca. Try using the following prompt;
> 
> > Below is an instruction that describes a task. Write a response that appropriately completes the request.
> > ### Instruction:
> > put your instruction here
> > ### Response:

That definitely helps with accuracy however it's a far cry from the gif shown in the readme:
![image](https://user-images.githubusercontent.com/9969843/226964951-e5753466-36cf-4e2d-9725-215dee2bb70e.png)

**Comment by ItsPi3141** [comment]
Not sure then, mine works fine. Although I rarely use 7B. I usually use 13 and 30B

**Comment by gururise** [comment]
Have you tried a model trained on the [Cleaned Dataset](https://github.com/gururise/AlpacaDataCleaned)?

Models trained on the original dataset sometimes have issues in their response.

**Comment by ItsPi3141** [comment]
> Have you tried a model trained on the [Cleaned Dataset](https://github.com/gururise/AlpacaDataCleaned)?

No, and sadly I can't because I don't have enough VRAM. If someone else fine-tunes it and creates the Lora weights, then I can merge it for alpaca.cpp/llama.cpp.
